machine_learning_hep/utils/hist.py

```diff
@@ -44,10 +44,13 @@ def get_nbins(hist, axis:int):
     return get_axis(hist, axis).GetNbins()
 
 
-# pylint: disable=too-many-branches
+# pylint: disable=too-many-branches, too-many-statements
 def project_hist(hist, axes: list, limits: dict[int, tuple[int]]):
     # TODO: add consistent suffix for projections
     # TODO: add option for uncertainties ???
+    if not hist:
+        print("Got null object")
+        return None
```

Comment:
This should behave like a proper library, that is not print stuff but raise an exception, probably `ValueError`.